packages/gatsby-plugin-typescript/src/gatsby-node.js

```diff
@@ -19,7 +19,10 @@ module.exports.modifyWebpackConfig = ({ config }, { compilerOptions }) => {
   const opts = { compilerOptions: copts, transpileOnly: true }
   config.loader(`typescript`, {
     test,
-    loader: `ts-loader?` + JSON.stringify(opts),
+    loaders: [
+      `babel?${JSON.stringify(config._loaders.js.config.query)}`,
```

Comment:
Just for information:

```javascript
// During a `yarn build` on my starter
config._loaders.js.config.query = {
          "plugins": [
            "D:\\Projets\\gatsby-starter\\node_modules\\gatsby\\dist\\utils\\babel-plugin-extract-graphql.js",
            "D:\\Projets\\gatsby-starter\\node_modules\\babel-plugin-add-module-exports\\lib\\index.js",
            "D:\\Projets\\gatsby-starter\\node_modules\\babel-plugin-add-module-exports\\lib\\index.js",
            "D:\\Projets\\gatsby-starter\\node_modules\\babel-plugin-transform-object-assign\\lib\\index.js"
          ],
          "presets": [
            "D:\\Projets\\gatsby-starter\\node_modules\\babel-preset-es2015\\lib\\index.js",
            "D:\\Projets\\gatsby-starter\\node_modules\\babel-preset-stage-0\\lib\\index.js",
            "D:\\Projets\\gatsby-starter\\node_modules\\babel-preset-react\\lib\\index.js"
          ],
          "cacheDirectory": true
        }
```
I will try to only take graphql-extract plugin ;) (but tomorow, it's late ^^)